src/VS.Web.CG.Mvc/Templates/Identity/Pages/Error.cs.cshtml

```diff
@@ -0,0 +1,25 @@
+@inherits Microsoft.VisualStudio.Web.CodeGeneration.Templating.RazorTemplateBase
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Identity;
+
+namespace @(Model.Namespace).Areas.Identity.Pages.Account
+{
+    public class ErrorModel : PageModel
+    {
+        public string RequestId { get; set; }
+
+        public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);
+
+        [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+        public void OnGet()
+        {
+            RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier;
```

Comment:
If it's possible for Activity to be null, it should be null checked too, i.e.:
`Activity?.Current?.Id...`

Reply:
@javiercn / @danroth27, is `Activity` guaranteed to be not null over here?